app/views/users/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<h4><%= @user.username %></h4>
+
+<div>
+  <img src="<%= @user.avatar %>">
+</div>
+<div data-role="follow">
+  <button data-role="new-follow" value="<%= @user %>">follow</button>
+</div>
+<div data-role="unfollow" class="hidden">
+  <button data-role="unfollow">unfollow</button>
+</div>
+<h5><%= pluralize(@user.followers.count, 'follower') %></h5>
+<div class="followers">
+  <% @user.followers.each do |follower| %>
+    <%= render partial: 'user', locals: { user: follower } %>
+  <% end %>
+</div>
+
+<h5><%= @user.followees.count %> following</h5>
+<div class="followees">
+  <% @user.followees.each do |followee| %>
+    <%= render partial: 'user', locals: { user: followee } %>
```

Comment:
would this mean that if I go and click on a user to view their profile I would be able to see a list of everyone they follow? :cookie: